Header header
bool enable
float32 max_speed_percentage
